packages/clis/generator/commands/extra-labels.ts

```diff
@@ -0,0 +1,191 @@
+import fs from 'fs';
+import type { GeoJSON } from 'geojson';
+import path from 'path';
+import type { Argv, BuilderArguments } from 'yargs';
+import { logger } from '../logger';
+import { writeGeojsonFile } from '../write-geojson-file';
+import { resourcesDir, untildify } from './path-helpers';
+
+import type { LabelMeta } from '@truckermudgeon/map/types';
+import type { TargetLabel } from '../geo-json/extra-labels';
+import { LabelProducer } from '../geo-json/extra-labels';
+
+export const command = 'extra-labels';
+export const describe =
+  'Generates map labels GeoJSON from parser output and optional metadata';
+
+const metaDefaults = ['usa-labels-meta.json'];
```

Comment:
Nit:

```suggestion
const metaDefaults = ['usa-labels-meta.json'] as const;
```

(without `as const`, consumers can still change the contents of `metaDefaults` at runtime 😕).

Reply:
I tend to view mutable defaults as a good thing. But in this case, I don't think it matters either way. Adding `as const`.